src/lib/storage/table.cpp

```diff
@@ -163,7 +164,10 @@ bool Table::empty() const { return row_count() == 0u; }
 
 ChunkID Table::chunk_count() const { return ChunkID{static_cast<ChunkID::base_type>(_chunks.size())}; }
 
-ChunkOffset Table::max_chunk_size() const { return _max_chunk_size; }
+ChunkOffset Table::target_chunk_size() const {
```

Comment:
Even though reference tables can only have a size of `MAX_CHUNK_SIZE`, is there any way to determine the chunk size for reference tables now?

Reply:
Is this related to #1792? Why would a reference table have a "chunk size"?

Reply:
@Bouncner this is probably used for the cost models, right?